timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchEventIntegrationTest.java

```diff
@@ -26,6 +26,7 @@
 import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Iterables;
 import com.google.common.util.concurrent.Uninterruptibles;
+import com.palantir.atlasdb.encoding.PtBytes;
```

Comment:
Do we want to be a bit more aggressive here with our testing?

Reply:
I think any test we add will already be mostly by some other test. But given that this is the ultimate integration test, I would be fine with replicating/bundling some of them here.

Reply:
I think a large-scale random test might make sense.

Reply:
Thinking about this more, I would rather use this class for advanced testing using actual transactions once we have the necessary changes in Atlas. The two tests are added don't really fit the purpose of this class, but are good sanity checks.

Reply:
I think that makes sense! Plus, this api is not really used anyways until that happens, so it's fine for this to merge beforehand.